newrelic/hooks/datastore_elasticsearch.py

```diff
@@ -108,42 +108,62 @@ def _nr_wrapper_Elasticsearch_method_(wrapped, instance, args, kwargs):
 _elasticsearch_client_methods = (
     ("abort_benchmark", None),
     ("benchmark", _extract_args_index),
-    ("bulk", None),
+    ("bulk", _extract_args_index),
     ("clear_scroll", None),
+    ("close", None),
+    ("close_point_in_time", None),
     ("count", _extract_args_index),
     ("count_percolate", _extract_args_index),
     ("create", _extract_args_index),
     ("delete", _extract_args_index),
     ("delete_by_query", _extract_args_index),
+    ("delete_by_query_rethrottle", None),
     ("delete_script", None),
     ("delete_template", None),
     ("exists", _extract_args_index),
+    ("exists_source", _extract_args_index),
     ("explain", _extract_args_index),
+    ("field_caps", _extract_args_index),
     ("get", _extract_args_index),
     ("get_script", None),
+    ("get_script_context", None),
+    ("get_script_languages", None),
     ("get_source", _extract_args_index),
     ("get_template", None),
     ("index", _extract_args_index),
     ("info", None),
+    ("knn_search", _extract_args_index),
     ("list_benchmarks", _extract_args_index),
-    ("mget", None),
+    ("mget", _extract_args_index),
     ("mlt", _extract_args_index),
     ("mpercolate", _extract_args_body_index),
-    ("msearch", None),
-    ("mtermvectors", None),
+    ("msearch", _extract_args_index),
+    ("msearch_template", _extract_args_index),
+    ("mtermvectors", _extract_args_index),
```

Comment:
So I'm seeing function signature changes between version for example this `mtermvectors` was `def mtermvectors(self, index=None, doc_type=None, body=None, params=None):` in version `6.2.0` but was `def mtermvectors(self, body=None, index=None, doc_type=None, params=None, headers=None)` in version `7.11`. I'm also not seeing any tests to make sure we are actually pulling out the index and not something else. I only see tests to make sure the method is being wrapped which is problematic as well. This looks like a bit of a mess and it is a pre-existing mess too. *crying*